dali/test/python/test_pipeline.py

```diff
@@ -424,6 +424,22 @@ def define_graph(self):
             img_chw = img_chw_test
         assert(np.sum(np.abs(img_chw - img_chw_test)) == 0)
 
+def test_none_seed():
+    batch_size = 60
+
+    for i in range(50):
+        pipe = Pipeline(batch_size=batch_size, num_threads=2, device_id=0, seed=None)
+        with pipe:
+            coin = fn.random.uniform(range = (0.0,1.0))
+        pipe.set_outputs(coin)
+        pipe.build()
+        pipe_out = pipe.run()[0]
+        img_chw_test = pipe_out.as_array()
+        if i == 0:
+            img_chw = img_chw_test
+        else:
+            assert(np.sum(np.abs(img_chw - img_chw_test)) != 0)
```

Comment:
This test will require that we don't get the same seed in any of the iterations - I thought about the opposite (check that in N pipelines there's at least one different seed). I don't require that, but in that case, running it just 2 times would be enough.

Reply:
We want to avoid that. I remember a case when two pipelines received the same seed because they were launched one after another and that can be bad (when you have a DDP strategy launching multiple pipelines in parallel).